src/js/api-interface.js

```diff
@@ -10,18 +10,23 @@ const baseUrl = `${generateUrl("apps/cookbook")}/webapp`
 // Add a debug log for every request
 instance.interceptors.request.use((config) => {
     Vue.$log.debug(
-        `Making "${config.method}" request to "${config.url}"`,
+        `[axios] Making "${config.method}" request to "${config.url}"`,
         config
     )
     const contentType = config.headers[config.method]["Content-Type"]
     if (!["application/json", "text/json"].includes(contentType)) {
         Vue.$log.warn(
-            `Request to "${config.url}" is using Content-Type "${contentType}", not JSON`
+            `[axios] Request to "${config.url}" is using Content-Type "${contentType}", not JSON`
         )
     }
     return config
 })
 
+instance.interceptors.response.use(
+    (response) => Vue.$log.debug("[axios] Received response", response),
+    (error) => Vue.$log.warn("[axios] Received error", error)
+)
+
```

Comment:
This causes me a problem during loading the app. If I disable these lines, I can access the cookbook again.

The following at least solved the problem:
```suggestion
instance.interceptors.response.use(
    (response) => {
        Vue.$log.debug("[axios] Received response", response)
        return response
    },
    (error) => Vue.$log.warn("[axios] Received error", error) /* ... */
)

```

I fear this is a problem with promises and the modification of mine does not cover the error case.

Reply:
I'm sorry. I have a bad habit of not testing thoroughly enough.